src/blr/cli/bamtofq.py

```diff
@@ -0,0 +1,96 @@
+"Writes fq files from bam files"
+
+import logging
+import pysam
+
+from tqdm import tqdm
+import dnaio
+
+from blr import utils
+
+logger = logging.getLogger(__name__)
+
+
+def main(args):
+    logger.info(f"Starting analysis")
```

Comment:
```suggestion
    logger.info("Starting analysis")
```
This string does not have to be f-string.